ioos_qc/qartod.py

```diff
@@ -320,31 +320,28 @@ def climatology_test(config : Union[ClimatologyConfig, Sequence[Dict[str, Tuple]
 
 
 def spike_test(inp : Sequence[N],
-               thresholds : Tuple[N, N]
+               suspect_threshold: float,
+               fail_threshold: float
```

Comment:
Can you change these to `N` to document you can use a `Double` or really any numeric that can be compared to the `inp` using `>` and `<`?

Reply:
Good catch. Updated to use `N`.